README.md

```diff
@@ -15,7 +18,7 @@ See dedicated documentation [**here**](https://github.com/fish-quant/autofish/bl
 1. Download latest version of miniconda from [**here**](https://docs.conda.io/en/latest/miniconda.html) (can also be Python 3.X).
 2. Open Anaconda terminal and create dedicated environment: `conda create --name autofish python=3.7`
```

Comment:
Hey, please note that "Python 3.7 will stop getting security updates in June 2023.", if possible I would use 3.8 or 3.9 for new projects.

Reply:
Thanks Master @oeway .

Will update this accordingly :-)